app/connectors/ArrivalConnector.scala

```diff
@@ -38,6 +38,11 @@ class ArrivalConnector @Inject()(http: HttpClient, appConfig: AppConfig) {
     http.POSTString(url, message)(HttpReads.readRaw, implicitly, implicitly)
   }
 
+  def postToMessagesRoute(message: String, arrivalId: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] = {
```

Comment:
Since calls to downstream /messages is going to only get bigger, it may be worth considering an ArrivalMessageController

Reply:
:thumbsup:

The code could be refactored to have ArrivalMovementController and ArrivalMessageController